ci-operator/config/openshift/external-dns-operator/OWNERS

```diff
@@ -0,0 +1,21 @@
+approvers:
+  - knobunc
+  - Miciah
+  - frobware
+  - danehans
+  - sgreene570
+  - candita
+  - rfredette
+  - alebedev87
+  - miheer
+reviewers:
+  - knobunc
+  - Miciah
+  - frobware
+  - danehans
+  - sgreene570
+  - candita
+  - rfredette
+  - alebedev87
+  - miheer
+component: DNS
```

Comment:
Why is this file not autogenerated as the corresponding cluster-dns-operator file is?  https://github.com/openshift/release/blob/5d78b36108622172baae3ae73a6416a12bd4b730/ci-operator/config/openshift/cluster-dns-operator/OWNERS#L1-L2

Reply:
According to https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/, the initial owners file for a new repo should be manually created. I think the autogeneration comments will appear in future OWNERS files syncs.